runtime/api-channel.js

```diff
@@ -252,8 +252,6 @@ class PECOuterPort extends APIPort {
     super(messagePort, 'o');
 
     this.registerCall('Stop', {});
-    this.registerCall('DefineParticle',
```

Comment:
This doesn't seem like a lint change - is it an unrelated cl that snuck in?

Reply:
I think a lint error somewhere led me to realize this is unused.